slsim/Plots/plot_functions.py

```diff
@@ -129,3 +129,156 @@ def plot_montage_of_random_injected_lens(image_list, num, n_horizont=1, n_vertic
         left=None, bottom=None, right=None, top=None, wspace=0.0, hspace=0.05
     )
     return fig
+
+
+def plot_lightcurves(data, images=True):
```

Comment:
it might be better to have more specific inputs in this definition to make it focused on the plotting